apps/molgenis-components/src/components/tables/RefSideModal.vue

```diff
@@ -20,66 +21,74 @@
 </template>
 
 <script lang="ts" setup>
-import { ref, toRefs, watch } from "vue";
-import Client from "../../client/client";
+import { Ref, ref, toRefs, watch } from "vue";
+import { IColumn } from "../../Interfaces/IColumn";
 import { IRefModalData } from "../../Interfaces/IRefModalData";
 import { IRow } from "../../Interfaces/IRow";
+import { default as Client, default as client } from "../../client/client";
 import ButtonAction from "../forms/ButtonAction.vue";
 import MessageError from "../forms/MessageError.vue";
 import Spinner from "../layout/Spinner.vue";
 import { getPrimaryKey } from "../utils";
 import RefTable from "./RefTable.vue";
 import SideModal from "./SideModal.vue";
 
-const props = withDefaults(
+const properties = withDefaults(
   defineProps<{
     schema?: string;
     tableId: string;
-    label: string;
+    column: string;
```

Comment:
what do you mean? columnName? what is purpose?

Reply:
columnName is better, yes. Renamed.